ClinGen/clincoded
# Connects to #1320. Experimental scoring bug.

**Steps to test:**
---
1. Start a new Experimental curation in GCI.
2. Select the **Biochemical Function** option for _Experiment type_.
3. Select the **A. Gene(s) with same function implicated in same disease** option for _Please select which one (A or B) you would like to curate_.
4. Expect to see the _Select Status_ selection dropdown being **disabled** in the **Exprimental Data Score** panel at the bottom.
5. Select the checkbox for _Has this gene(s) been implicated in the above disease?_.
6. Expect to see the _Select Status_ selection dropdown being **enabled** in the **Exprimental Data Score** panel at the bottom.
---
7. Now select the **B. Gene function consistent with phenotype(s)** option for _Please select which one (A or B) you would like to curate_ .
8. Expect to see the _Select Status_ selection dropdown being **enabled** in the **Exprimental Data Score** panel at the bottom.
---
9. Select the **Expression** option for _Experiment type_.
10. Select the **A. Gene normally expressed in tissue relevant to the disease** option for _Please select which one (A or B) you would like to curate_.
11. Expect to see the _Select Status_ selection dropdown being **disabled** in the **Exprimental Data Score** panel at the bottom.
12. Select the checkbox for _Is the gene normally expressed in the above tissue?_.
13. Expect to see the _Select Status_ selection dropdown being **enabled** in the **Exprimental Data Score** panel at the bottom.
---
14. Now select the **B. Altered expression in Patients** option for _Please select which one (A or B) you would like to curate_.
15. Expect to see the _Select Status_ selection dropdown being **disabled** in the **Exprimental Data Score** panel at the bottom.
16. Select the checkbox for _Is expression altered in patients who have the disease?_.
17. Expect to see the _Select Status_ selection dropdown being **enabled** in the **Exprimental Data Score** panel at the bottom.
18. Deselect the checkbox for _Is expression altered in patients who have the disease?_.
19. Expect to see the _Select Status_ selection dropdown being **disabled** again.
---
20. Switch back to the **A. Gene normally expressed in tissue relevant to the disease** option for _Please select which one (A or B) you would like to curate_.
21. Expect to see the _Select Status_ selection dropdown being **disabled** in the **Exprimental Data Score** panel at the bottom.
22. Expect to see the checkbox for _Is the gene normally expressed in the above tissue?_. being deselected.
---
23. Switch back to the **Biochemical Function** option for _Experiment type_.
24. Select the **A. Gene(s) with same function implicated in same disease** option for _Please select which one (A or B) you would like to curate_ again.
25. Expect to see the checkbox for _Has this gene(s) been implicated in the above disease?_ being disabled.
26. Expect to see the _Select Status_ selection dropdown being **disabled**.
